lib/exception_handling.rb

```diff
@@ -214,21 +203,41 @@ def write_exception_to_log(ex, exception_context, timestamp)
       end
     end
 
+    #
+    # Send notifications to configured external services
+    #
+    def send_external_notifications(exception_info)
+      results = {}
+      if honeybadger?
```

Comment:
so that we don't make the end user of the gem juggle what `nil` vs `false` means, how do we feel about inverting this to `failed_to_notify_honeybadger`?

Reply:
Maybe it's better to pass the explicit state through?
```:success, :failure, :skipped```

Reply:
yea, lets do that then. and call it `honeybadger_status`?